File: TestTask.Domain/Application.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace TestTask.Domain
{
    public class Application
    {
        public int Id { get; set; }
        public string FIOApplicant { get; set; }
        public string PassportInfo { get; set; }
        public string DateBirth { get; set; }
        public string FIOChildOfApplicant { get; set; }
        public string BirthCertificate { get; set; }
        public string DateBirthCildOfApplicant { get; set; }
    }
}

File: TestTask/ClientApp/src/app/main/application.ts
export class Application{
    constructor(
        public Id: number,
        public FIOApplicant: string,
        public PassportInfo: string,
        public DateBirth: Date,
        public FIOChildOfApplicant: string,
        public BirthCertificate: string,
        public DateBirthCildOfApplicant: Date,
        public AvailabilityOfBenefitsv: string,
        public BankAccountNumber: number){}
}